setup.py

```diff
@@ -0,0 +1,54 @@
+"""Modules"""
+import asyncio
+from setup import installation, post_installation, utils
+
+"""Display initial data"""
+utils.display_markdown('# Talawa API')
+with open('./setup/data/About.md', encoding="utf-8") as about:
```

Comment:
OK, now I see the purpose of the markdown files. Please rename `data/` to `markdown/` so it is clearer.